tasks/src/object-detection/data.ts

```diff
@@ -65,8 +65,8 @@ const taskData: TaskDataCustom = {
 			id:          "kadirnar/yolov8",
 		},
 		{
-			description: "YOLOv6: a single-stage object detection framework dedicated to industrial applications.",
-			id:          "kadirnar/yolov6",
+			description: "TorchYolo: YOLO Series Object Detection and Track Algorithm Library.",
```

Comment:
Can you write a more general explanation for this? 
Maybe something like:
"An application that shows multiple cutting edge techniques for object detection and tracking."

Reply:
Thank you. I fixed it.